lib/test/cdo/contact_rollups/test_pardot_v2.rb

```diff
@@ -150,4 +150,21 @@ def test_submit_batch_request_with_errors
 
     assert_equal expected_errors, PardotV2.submit_batch_request('a_pardot_endpoint', prospects)
   end
+
+  def test_delete_in_non_production_fails
```

Comment:
Want to add another test or two here for when this method gets past the two return statements, could use input on how to best do that!

Reply:
These 2 tests are for negative cases; you can add 1 more positive test. For example, a request to delete prospect is sent and we want to verify that the destination URL is correct.